packages/calcite-components/src/components/list-item/list-item.tsx

```diff
@@ -51,6 +51,7 @@ import {
  * @slot content - A slot for adding non-actionable, centered content in place of the `label` and `description` of the component.
  * @slot content-end - A slot for adding non-actionable elements after the label and description of the component.
  * @slot actions-end - A slot for adding actionable `calcite-action` elements after the content of the component.
+ * @slot content-bottom - A slot for adding content below the label and description of the component.
```

Comment:
Awesome! Small recommended tweak and consistency with backticks in to differentiate the props:

```
A slot for adding content below the component's `label` and `description`.
```